PartyWebApp/Models/Performer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PartyWebApp.Models
+{
+    public class Performer
+    {
+        public int id { get; set; }
```

Comment:
partyId it's better - because it's primary key auto

Reply:
So just Id, not performerId, because the entire object is referred to as Performer.
You'll then have to refer to the object id like this => performer.performerId instead of performer.Id